topics/statistics/tutorials/genomic-llm-finetuning/tutorial.md

```diff
@@ -702,17 +715,6 @@ eval_steps_per_second          9.620000
 epoch                          3.000000
 ```
 
-> <question-title></question-title>
```

Comment:
I forgot to do it. But the idea was to move the above output and comment it in a question format